02-electric/electric-section-three/electric-section-three.css

```diff
@@ -1,65 +1,327 @@
+.electric-section-three-part-two-copy-container,
+.electric-section-three-part-two-subtitle,
+.electric-section-three-part-two-paragraph-container,
+.electric-section-three-part-two-paragraph,
+.electric-section-three-part-three-container,
+.electric-section-three-part-three-image-container,
+.electric-section-three-part-three-copy-container,
+.electric-section-three-part-three-subtitle,
+.electric-section-three-part-three-paragraph-container,
+.electric-section-three-part-three-paragraph{
+    border:var(--temporary-border)
+}
 
-    .electric-section-three-container{
-    padding: 15px;
-    }
-    
-    .electric-section-three-container >*{
-        border: solid  1px black;
-    }
-    
-    .electric-section-three-subcontainer{
-        display: grid;
-        grid-template-columns: 1fr;
-        justify-content: center;
-        row-gap: 15px;
-        padding: 25px
-      
-      }
+.electric-section-three-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  padding: var(--container-padding-mobile);
   
-    .electric-section-three-subcontainer > *{
-          border: solid  1px black;
-      }
-      
-      .electric-section-three-img-container{
-          grid-column: 1/2;
-          
-      }
-
-    .electric-section-three-img{
-      display: block;
-        width: 100%;
-    }
-      
-    .electric-section-three-title{
-        grid-column: 1/2;
-         text-align: center;
-      }
-
-    .electric-section-three-content{
-          grid-column: 1/2;
-         text-align: justify;
-      }
-      
-      .electric-section-three-small-content-container{
-        grid-column: 1/2;
-        display: grid;
-        grid-template-columns: 1fr 3fr;
-      }
-      .electric-section-three-small-img{
-        grid-column: 1/2;
-        justify-self: center;
-        align-self: center;
-        width: 100%;
-      }
-      
-      .electric-section-three-small-prg{
-        grid-column:2/3;
-
-      }
-
-    .electric-section-three-btn{
-        grid-column: 1/2;
-        width:90px;
-        height: 40px ;
-        justify-self: center;
-      }
\ No newline at end of file
+}
+
+@media (min-width:500px) {
+  .electric-section-three-container {
+    padding: var(--container-padding-desktop);
+  }
+}
+
+.electric-section-three-subcontainer {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-areas:
+  "electric-section-three-title"
+  "electric-section-three-part-one-container"
+  "electric-section-three-part-two-container"
+  "electric-section-three-part-three-container";
+  justify-content: center;
+  row-gap: var(--subsection-container-row-gap-mobile);
+  column-gap:none;
+  padding: var(--subcontainer-padding-mobile);
+  border: solid 1px black;
+  background-color: var(--level1-background-color);
+}
+
+@media (min-width:500px){
+  .electric-section-three-subcontainer{
+    grid-template-columns: repeat(2,1fr);
+    grid-template-areas:
+    "electric-section-three-title electric-section-three-title"
```

Comment:
this will need changing to "electric-section-three-title-container electric-section-three-title-container"

Reply:
done